staging/src/k8s.io/code-generator/cmd/validation-gen/validators/union.go

```diff
@@ -161,7 +162,15 @@ func (c *unionDeclarativeValidator) ExtractValidations(t *types.Type, comments [
 			}
 		}
 	}
-	for unionName, u := range unions {
+
+	// Sort the keys for stable output.
+	keys := make([]string, 0, len(unions))
+	for k := range unions {
+		keys = append(keys, k)
+	}
+	slices.Sort(keys)
+	for _, unionName := range keys {
+		u := unions[unionName]
```

Comment:
I had https://github.com/jpbetz/kubernetes/pull/26 sitting around, but this is simpler. I'll close mine.